docs/docs/reference/cli/testkube_create_test.md

```diff
@@ -45,12 +46,13 @@ testkube create test [flags]
       --prerun-script string                       path to script to be run before test execution
       --schedule string                            test schedule in a cronjob form: * * * * *
       --scraper-template string                    scraper template file path for extensions to scraper template
+      --secret-env stringToString                  secret envs in a form of secret_key1=secret_name1 passed to executor (default [])
   -s, --secret-variable stringToString             secret variable key value pair: --secret-variable key1=value1 (default [])
       --secret-variable-reference stringToString   secret variable references in a form name1=secret_name1=secret_key1 (default [])
       --source string                              source name - will be used together with content parameters
       --test-content-type string                   content type of test one of string|file-uri|git
       --timeout int                                duration in seconds for test to timeout. 0 disables timeout.
-  -t, --type string                                test type (defaults to postman/collection)
+  -t, --type string                                test type
```

Comment:
should be postman?

Reply:
if it cannot detect type it should give an error